.github/workflows/test.yml

```
@@ -0,0 +1,85 @@
+name: Test Workflow
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, edited, labeled, workflow_dispatch]
+  workflow_dispatch:
+      inputs:
+        dry-run:
+          description: 'Run tests but do not create issues {true,false}'
+          required: true
+          default: 'true'
+        notify-id:
+          description: 'Issue notification {github id}'
+          required: false
+        vendor-type:
+          description: 'Vendor type {all,partner,redhat,community}'
+          required: true
+          default: 'all'
+        software-name:
+          description: 'Software Name'
+          required: true
+        software-version:
+          description: 'Software Version'
+          required: true
+
+
+jobs:
+  workflow-test:
+    name: Workflow Test
+    runs-on: ubuntu-20.04
+    if: github.event.pull_request.draft == false
+
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v2
```

```suggestion
        uses: actions/checkout@v2
        with:
            token: ${{ github.event.inputs.bot-token }}
            fetch-depth: 0
```

`token` is used otherwise the git push will fail with 403 access denied error: https://github.com/zonggen/charts/runs/3206243656?check_suite_focus=true#step:6:357

`fetch-depth` is used to disable shallow clone error: https://github.com/zonggen/charts/runs/3211479566?check_suite_focus=true#step:6:358

---

Fixed